### PR TITLE
Added Guus to team page.

### PR DESCRIPTION
**Pull request by GuusAvis**

### CODE REVIEW

File: _data/team_members.yml [hunk 1]
```diff
@@ -8,6 +8,17 @@
     - Master, Sciences de la matière, <a href="https://www.ens-lyon.fr">ENS Lyon, France</a>
     - PhD, Physics, <a href="https://quantuminstitute.yale.edu/">Yale Quantum Institute</a>
     - Postdoc, <a href="https://www.rle.mit.edu/qp">MIT/Harvard</a>
+- name: Guus Avis
+  photo: Guus1.jpg
+  info: Postdoctoral Researcher
+  time: started August 2023
```
Comment (GuusAvis) [inline _data/team_members.yml:14]: It seems this field does not actually make it to the final web page, but I still added it for consistency.
Is this on purpose or an oversight @Krastanov ?

File: _data/team_members.yml [hunk 2]
```diff
@@ -8,6 +8,17 @@
...
+  email: gavis@umass.edu
+  website: https://scholar.google.com/citations?user=ZOhYdlkAAAAJ&hl=en&oi=ao
```
Comment (GuusAvis) [inline _data/team_members.yml:16]: I tried to compile this as a link that says "Google scholar" with a hyperlink to the actual page, but that doesn't work (either using markdown or html syntax). It seems the field is already wrapped by some html tags by Franklin or something.

Any idea how to make the text and the hyperlink different from one another @Krastanov ? If it's hard we can just leave it as is, but the google scholar link looks kinda ugly.

**Comment by Krastanov** [comment]
I pushed a commit that should address this. Check it out and feel free to modify it

File: _data/team_members.yml (hunk 1 above)
Comment (Krastanov) [reply _data/team_members.yml:14]: an oversight, but we do not need to worry about it in this PR

**Pull request merged**